notice_and_comment/static/regulations/css/less/typography-custom.less

```diff
@@ -21,26 +21,31 @@ h6 {
 }
 
 h1 {
-    font-family: 'Source Sans Pro', Georgia, sans-serif;
-    font-weight: 300;
-    font-size: 44px;
-    line-height: 50px;
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
-h2, h3.section-title {
-    font-size: 34px;
-    line-height: 40px;
-    margin: 20px 0;
+h2 {
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
 h3 {
-    font-size: 24px;
-    line-height: 29px;
+  font-size: 34px;
+  line-height: 40px;
+}
+
+h4 {
+  font-size: 24px;
+  line-height: 29px;
+  text-transform: none;
```

Comment:
for the sake of consistency I'm going to add the `uppercase` to h5 here even though it isn't over-riding anything. h5's in N&C should be uppercase. 

I think in the summer when we unify/refactor these styles we'll have to take a look at type hierarchy and make the call of whether or not we even want to declare font casing in the h* styles (I lean towards no)